prep-css-display/index.html

```diff
@@ -1,12 +1,40 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title>CSS Display Properties</title>
-  <style>
-
-  </style>
+  <title>prep-example</title>
 </head>
 <body>
-
+    <h1>I would like to build</h1>
+  <p>
+    <!-- write your app ideas here 👇 -->
+    I would like to build apps to help someone who want to learn how to play the guitar!
+  </p>
+  <div style="border: 4px solid blue;"> I am a div and by default I am block, which means I take up all the space I can
+    horizontally.</div>
+  <div>
+    I am a div but I have <span style="border: 4px solid green;height: 100px;width: 100px;" >span elements inside me</span>.
+    Span elements are <span style="border: 4px solid green;height: 100px;width: 100px;">display inline by default</span>,
+    so they only take up as much as space as they need.
+  </div>
+  <div>
+    <button style="border:4px solid red;height: 40px;width: 200px;">Hello!</button>
+    <button style="border:4px solid red;height: 40px;width: 200px;">Buttons are</button>
+    <button style="border:4px solid red;height: 40px;width: 200px;">Inline Block</button>
+    <button style="border:4px solid red;height: 40px;width: 200px;">By Default</button>
+  </div>
+<ul style="border:4px solid purple;">
+  <li style="border:4px solid magenta;display: ;">
+    Home
+  </li>
+  <li style="border:4px solid magenta;display: ">
+    Products
+  </li>
+  <li style="border:4px solid magenta;display:">
+    About Us
+  </li>
+  <li style="border:4px solid magenta;display:;">
+    Contact Us
+  </li>
```

Comment:
Hi Justin, for this exercise, we want to write all of our styling in the <style> element that was provided in the starter file instead of inline on each individual element.